### ISSUE TITLE
[BUG] `htmlParser: selector` ignored when using `settings.json`

### ISSUE THREAD

**Issue by RickTorresJr** [markmap/markmap-vscode#55]
- [X ] I have searched for existing issues that already reported this problem and found none
- [ X] The bug for the VSCode plugin
  
- [ ] The bug is also present in [the web version of VSCode](https://vscode.dev) or GitHub Codespaces

**Describe the bug**

I'm trying to use the following equivalent options as the defaults for `settings.json`:

```
---
markmap:
  initialExpandLevel: 4
  htmlParser:
    selector: 'h1, h2, h3, h4, h5, h6'
---
```

The `htmlParser` works when using it as front matter but not in `settings.json`. `initialExpandLevel` works.

**To Reproduce**

![image](https://github.com/user-attachments/assets/804286fc-7740-4113-8b45-4cbdec430272)

Add the following to extension settings under Markmap: Default Options

```
{
    "initialExpandLevel": 4,
    "htmlParser": {
        "selector": "h1, h2, h3, h4, h5, h6"
    }
}
```

**Expected behavior**

Same behavior when using front matter (only show Markdown headers -- do not use `*`)

```
---
markmap:
  initialExpandLevel: 4
  htmlParser:
    selector: 'h1, h2, h3, h4, h5, h6'
---
```

**Actual behavior**

Only `initialExpandLevel` works and `htmlParser` does not work.
**Environment:**

- OS: Windows 10

**Additional context**

**Comment by gera2ld** [comment]
Fixed in 0.1.6

**Comment by RickTorresJr** [comment]
Confirmed fixed. Thank you @gera2ld
